lib/system/database.rb

```diff
@@ -158,7 +158,7 @@ def adapter_module
         # permissions.
         def create
           if ENV['ORACLE_SYSTEM_PASSWORD'].present?
-            Rails.logger.warn("Oracle's SYSTEM user will create/update a non-SYSTEM user and grant it permissions")
+            Logger.new(STDOUT).warn("Oracle's SYSTEM user will create/update a non-SYSTEM user and grant it permissions")
```

Comment:
nitpick, prefet `$stdout` [1]. But also shouldn't we use `stderr`? idk where default log of rails goes but I'm used for shell apps to have log to stderr instead. Only the processable output to stdout. 

[1] https://docs.rubocop.org/rubocop/cops_style.html#styleglobalstdstream

Reply:
Updated in [ed38bdb](https://github.com/3scale/porta/pull/3195/commits/ed38bdb40a2ec8e3ecd40d7fffc8d32aa5719ed0).